RELEASES.md

```diff
@@ -86,3 +86,122 @@ For that we need to take some precautions:
+
+```sh
+npx changeset;
+# Select the package and set as major version
+# Commit the changeset
+git add .changeset; git commit -m 'chore: add changeset for [package name]'
+
+npx changeset version;
+# This will update the pre.json and the package.json file for the selected package.
+# If the package is already in alpha, it will update the version number after -alpha, e.g. alpha.0 > alpha.1
+# If the package wasn't in alpha yet, the version will be updated to the next major version and -alpha.0 added at the end.
+```
+
+#### Moving to stable version
+
+After having some pre release changsets created like on the previous step, you can run the following to look how it would look like when moving to stable.
+
+```sh
+npx changeset pre exit;
+# This updates the pre.json file to mark it as exiting prereleases mode
+
+# To test how it would update the changelogs and how it would behave as in main branch, replace the .changeset/config.json file with the one from the main branch, before running changeset version
+
+npx changeset version;
+# This will remove the pre.json file, and update all package.json files that need to be updated to the version without the -alpha tag.
+
+```
+
+### Changeset configuration
+
+To make it possible we take advantage of changeset feature of prereleasing, to make it possible we have the [.changeset/pre.json](./.changeset/pre.json) which specifies which tag to use and the initial version of the packages, which is used to calculate the alpha version, for example:
+
+```
+initial version: 4.61.2
+
+For that initial version, each type of changeset would update as following:
+patch: 4.61.3-alpha.0
+minor: 4.62.0-alpha.0
+major: 5.0.0-alpha.0
+
+If the initial version is already on the alpha tag, either version would bump the alpha.[x]
+```
+
+_In our case we want all changes to be major so we work on the following major release._
+
+**Config file explanation**
+
+The main differences on the config file are the following:
+
+```js
+{
+  "changelog": false // This is set so the changelog of the packages is not updated
+  "ignore": [] // We don't want to ignore any package on the next branch
+  "fixed": [] // We don't want to have fixed or linked versions of alpha releases
+  "baseBranch": "next", // The branch which changesets uses to check versions
+  "bumpVersionsWithWorkspaceProtocolOnly": true, // Avoid bumping dependencies on other packages
+}
+```
+
+Notes:
+
+- The changelog is not updated because when we exit the prerelease state the changeset will compile all the changesets and apply onto the new version. eg. 5.0.0 will have all the changesets of the 5.0.0-alpha.\*
+- The `bumpVersionsWithWorkspaceProtocolOnly` makes it possible to avoid bumping packages on the packages that are dependent on the pacakge being updated, this is because we want alpha packages to depend on stable packages, rather than have all of them depending on alpha packages. ie. we would make all packages bump to an alpha core in case we update it.
```

Comment:
```suggestion
> [!NOTE]
> 
> - The changelog is not updated because after exiting the prerelease state, all the changesets will be compiled and applied to the new version. For example, `5.0.0` will have all the changesets of the `5.0.0-alpha.\*`
> - The `bumpVersionsWithWorkspaceProtocolOnly` prevents bumping packages' versions after a package they depend on is updated. This is because we want alpha packages to depend on stable packages. We would bump all packages to an alpha core if we allow it.
```